lang/system/sv.xml

```diff
@@ -24,41 +36,88 @@
 		</help>
 	</main_menubar>
 
+	<about name="Om">
+		<vita3k>Vita3K: en PS Vita/PS TV-emulator. Världens första funktionella PS Vita/PS TV-emulator.</vita3k>
+		<about_vita3k>Vita3K är en experimentell PlayStation Vita/PlayStation TV-emulator skriven I C++ för operativsystemen Windows, Linux, macOS samt Android och är baserad på öppen källkod.</about_vita3k>
+		<special_credit>Speciallt tack: Vita3K-ikonen designades av:</special_credit>
+		<github_website>Om du är intresserad att bidra, se vår:</github_website>
+		<vita3k_website>Besök vår webbplats för mer info:</vita3k_website>
+		<ko-fi_website>Om du vill hjälpa oss kan du göra en donation eller prenumerera på vår:</ko-fi_website>
+		<vita3k_staff>Vita3K-teamet</vita3k_staff>
+		<developers>Utvecklare</developers>
+		<contributors>Bidragsgivare</contributors>
+		<supporters>Supporters</supporters>
+	</about>
+
 	<app_context>
-		<manual>Handbok</manual>
+		<check_app_state>Kontrollera programtillstånd</check_app_state>
+		<copy_vita3k_summary>Kopiera Vita3K-sammandrag</copy_vita3k_summary>
+		<open_state_report>Öppna tillståndsrapport</open_state_report>
+		<create_state_report>Skapa tillståndsrapport</create_state_report>
+		<update_database>Uppdatera databasen</update_database>
+		<copy_app_info>Kopiera programinfo</copy_app_info>
+		<name_and_id>Namn och titel-id</name_and_id>
+		<app_summary>Programsammandrag</app_summary>
+		<custom_config>Anpassad konfig</custom_config>
+		<create>Skapa</create>
+		<edit>Redigera</edit>
+		<remove>Ta bort</remove>
+		<open_folder>Öppna mapp</open_folder>
+		<addcont>DLC</addcont>
+		<license>Licens</license>
+		<shaders_cache>Shaders-cache</shaders_cache>
+		<shaders_log>Shaders-logg</shaders_log>
+		<manual>Manuell</manual>
 		<update>Uppdatera</update>
 		<update_history>Uppdateringshistorik</update_history>
 		<history_version>Version {}</history_version>
 		<information>Information</information>
 		<delete>
-			<app_delete>Det här programmet och alla tillhörande data, inklusive sparade data, kommer att raderas.</app_delete>
-			<saved_data_delete>Vill du radera dessa sparade data?</saved_data_delete>
+			<app_delete>Detta program och all relaterad data, inklusive sparat data, kommer att tas bort.</app_delete>
+			<app_delete_description>Borttagning av ett program kan ta en stund,
+beroende på dess storlek och din hårdvara.</app_delete_description>
+			<addcont_delete>Vill du ta bort detta tilläggsdata?</addcont_delete>
+			<license_delete>Vill du ta bort denna licens?</license_delete>
+			<saved_data_delete>Vill du ta bort detta sparade data?</saved_data_delete>
 		</delete>
 		<info>
 			<eligible>Möjligt</eligible>
 			<ineligible>Inte möjligt</ineligible>
 			<level>Nivå</level>
 			<name>Namn</name>
-			<trophy_earning>Vinna trofé</trophy_earning>
-			<parental_controls>Föräldrakontroll</parental_controls>
-			<updated>Uppdaterat</updated>
+			<trophy_earning>Tjäna troféer</trophy_earning>
```

Comment:
Revert this line, all string already set using officiel translation, no touching it